### DISCUSSION TITLE
Deployment issue on digitalocean

### DISCUSSION THREAD

**Discussion by jake1808** [medusajs/medusa#5020] [Q&A] [answered]
Hello, I recently deployed a medusa backend on digital ocean and followed the guide on the docs. I manage to successfully deplo and I'm able to hit the `store/products` endpoint but when I use the admin panel  `/app` endpoint all my network calls are in a status of 401.

**Comment by pevey** [comment, marked as answer]
Can you post the errors from the browser console?  Is there anything about CORS errors?

**Reply by jake1808** [reply]
![image](https://github.com/medusajs/medusa/assets/51802968/af40cd16-829c-4c9b-814f-b9c8a3392c72)
`Request URL:
https://micmar-suz5s.ondigitalocean.app/admin/store/
Request Method:
GET
Status Code:
401
Remote Address:
162.159.140.98:443
Referrer Policy:
strict-origin-when-cross-origin
Access-Control-Allow-Credentials:
true
Cache-Control:
private
Cf-Cache-Status:
MISS
Cf-Ray:
8061e15aacd54eb2-JNB
Date:
Wed, 13 Sep 2023 16:48:53 GMT
Server:
cloudflare
Vary:
Origin, Accept-Encoding
Www-Authenticate:
Bearer realm="Users"
X-Do-App-Origin:
97396561-7dda-4818-bebb-10abebe94cf5
X-Do-Orig-Status:
401
X-Powered-By:
Express
:authority:
micmar-suz5s.ondigitalocean.app
:method:
GET
:path:
/admin/store/
:scheme:
https
Accept:
application/json
Accept-Encoding:
gzip, deflate, br
Accept-Language:
en-US,en;q=0.9
Cookie:
connect.sid=s%3AC6eFtctMOXptticV844sOSYJYU11K7mk.g%2Fkr7O1RseaW%2BrqSQI3d0GC3LVQZiqwWt9UTc4mtruA
Referer:
https://micmar-suz5s.ondigitalocean.app/app/a/orders
Sec-Ch-Ua:
"Chromium";v="116", "Not)A;Brand";v="24", "Google Chrome";v="116"
Sec-Ch-Ua-Mobile:
?0
Sec-Ch-Ua-Platform:
"Windows"
Sec-Fetch-Dest:
empty
Sec-Fetch-Mode:
cors
Sec-Fetch-Site:
same-origin
User-Agent:
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36`
![image](https://github.com/medusajs/medusa/assets/51802968/eb3b9996-c1f4-4a68-ad5d-6fb164a0dbe3)

**Reply by pevey** [reply]
That is not the console

**Reply by jake1808** [reply]
![image](https://github.com/medusajs/medusa/assets/51802968/2d1493ea-5b47-4350-ad58-91da3fd227a5)
sorry about that here is my console.

**Reply by jake1808** [reply]
This is my first deploying medusa how ever I have just been experimenting with it in dev. Thank you @pevey I will join discord and seek further advice from there.

**Reply by pevey** [reply]
It doesn't say anything about CORS, so it looks like that is not the issue. (It commonly is.)

Can you access you backend directly by going to BACKEND_URL:/store/products, for instance?  

There are more people in the Medusa Discord who can help you troubleshoot.  You might want to repost this there for better response.

**Reply by jake1808** [reply]
and could you please share a link to the discord thank you.
